Get_Repo_Details/get_repo_details.py

```diff
@@ -0,0 +1,90 @@
+# Script to GET Repo details from user
+
+import json
+import requests
+import os
+import pprint
+import fire
+
+# constants useful for the script
+BASE_URL = "https://api.github.com/graphql" # Github base url
+GITHUB_TOKEN = os.environ['GITHUB_TOKEN']  # Github token environment variable
+
+def getQuery(repoName, userName):
+  '''
+  Returns the query string that will fetch data from the API.
+  Gets the number of collaborators, number of stars, 
+  number of pull requests, and total commits of a particular repository.
+
+  Args:
+    repoName : Name of the repository from which to fetch data
+    userName : Name of the owner of the repository
+  
+  Return:
+    query : Query string which will be used to query the API endpoint
+  '''
+  query = """query {{
+    repository(name: \"{0}\", owner: \"{1}\") {{
+      collaborators {{
+        totalCount
+      }}
+      pullRequests {{
+        totalCount
+      }}
+      stargazers {{
+        totalCount
+      }}
+      object(expression: "master") {{
+        ... on Commit {{
+          history {{
+            totalCount
+          }}
```

Comment:
This is a totally unnecessary function definition so initialize a string. remove it and  keep it as a simple variable

Reply:
Removed it